src/parse.js

```diff
@@ -12,8 +12,22 @@ const STATE_MODIFIER = Symbol("modifier");
 const STATE_FUNCTION = Symbol("function");
 const STATE_NAME = Symbol("name");
 
-export function parseCell(input, {globals} = {}) {
-  const cell = CellParser.parse(input);
+
+export function parseCell(input, {globals, tag, raw} = {}) {
+  return tag != null
+    ? parseTemplateCell(input, {globals, tag, raw})
+    : parseJavaScriptCell(input, {globals});
+}
+
+function parseJavaScriptCell(input, options) {
+  return finishCell(CellParser.parse(input), input, options);
+}
+
+function parseTemplateCell(input, {tag, raw, ...options}) {
+  return finishCell(TemplateCellParser.parse(input, {tag, raw}), input, options);
+}
+
+function finishCell(cell, input, {globals}) {
```

Comment:
Nice -- in order to avoid populating `tag` and `raw` for normal JavaScript cells, I added a conditional. Let me know if you prefer all cell types to have these properties and I'll remove it and record the snapshot changes 👍